src/Symfony/Component/HttpKernel/DataCollector/DumpDataCollector.php

```diff
@@ -131,6 +131,7 @@ public function dump(Data $data)
 
         if ($this->dumper) {
             $this->doDump($data, $name, $file, $line);
+            $this->isCollected = true;
```

Comment:
Instead we could tweak https://github.com/symfony/symfony/pull/26675/files#diff-b043f95afd7daa1be0bdd864620c510dR70 to check if `$this->dumper` is set. But I find it more expressive as is. Let me know.

Reply:
if we can achieve the same without mutating any state, that may be better?

Reply:
if we can achieve the same without mutating any state, that may be better?

Reply:
Alright